### DISCUSSION TITLE
Is it possible to run this project in NOOBS?

### DISCUSSION THREAD

**Discussion by slipperystairs** [riffnshred/nhl-led-scoreboard#392] [General]
The step by step guide states to use Raspberry Pi OS Lite to achieve the best performance . I was curious if anyone has run this project on NOOBS? If so, did you run into any issues?

Thanks!

**Comment by pjockey** [comment]
The OS needs to be a "minimal" OS. No GUI and works headless. I don't know enough about NOOBS and whether or not you can install it as a minimal OS (no GUI).

**Comment by falkyre** [comment]
Based on what I've read, all NOOBS provides is the way to step a new user through getting an OS onto your raspberry pi.  It is a smaller bootable image that will prompt you to pick your OS to install, configure your wifi etc.  Then it becomes a bootloader that sits in the background of your boots to allow you to boot into it and perform some maintenance functions if you want to.  So in the grand scheme of this project, all it really does is remove the process of creating your Raspberry Pi image on an SD card and setting up the wifi.  You still need to do everything else once the OS is installed. It might be useful for those who are nervous about editing files to get wifi working etc.
